src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -1154,7 +1154,60 @@ public final class JavadocTokenTypes {
      */
     public static final int P_HTML_TAG_NAME = JavadocParser.P_HTML_TAG_NAME;
 
-    /** List item tag name. */
+    /**
+     * List item tag name.
+     *
+     * <p><b>Example:</b></p>
+     * <pre>{@code
+     *  <ol>
+     *    <li>banana</li>
+     *  </ol>
+     *  }</pre>
+     *  <b>Tree:</b>
+     *  <pre>
+     *  {@code
+     *  JAVADOC -> JAVADOC
+     *   |--TEXT -> /**
+     *   |--NEWLINE -> \r\n
+     *   |--LEADING_ASTERISK ->  *
+     *   |--TEXT ->
+     *   |--HTML_ELEMENT -> HTML_ELEMENT
```

Comment:
based on @romani 's suggestion in a different PR, I would recommend to remove other tags above `HTML_ELEMENT` to make it compact